src/program/ipfix/probe/probe.lua

```diff
@@ -95,10 +91,6 @@ function run (args)
       c = function (arg)
          collector_ip = arg
       end,
-      -- TODO: this should probably be superceded by using ARP
-      M = function (arg)
-         collector_mac = arg
-      end,
```

Comment:
These options should be removed too from the list of accepted options. Line 115:

```lua
args = lib.dogetopt(args, opt, "hsD:i:o:p:m:a:c:M:", long_opts)
```